File: JAVASCRIPTTODO/13-dic/ciclos.js
/*

Ciclos - Bucles - Loops

let condicion = 1; valor inicial

while(condicion){
    tu codigo
}

*/

let condicion = 10;

while(condicion<=10){
    console.log(condicion);
    //condicion = condicion + 1;
    condicion++;
}

/* let numero = 1;

numero++;
numero++;
numero++;
numero--;
numero--;
numero--;
console.log(numero); */

/*
do while
*/

let numero = 10;
do{
    console.log(numero);
    numero++;
}while(numero <=10);

/*
for
*/

for (let control = 0; control <=10; control++) {
    console.log(control);
    
}

const comidas = ['Pozole', 'Tacos', 'Tamales', 'Tortas'];
for (let i = 0; i <= comidas.length; i++) {
    console.log(comidas[i]);
    
}

File: JAVA/JavaBasicos/src/operadores/Ciclos.java
package operadores;

public class Ciclos {
	public static void main(String[] args) {
		//while
		
		int control = 0;
		
		while(control <10) {
			System.out.println("Control = " + control);
			control ++;
		}
		
		//do while
		control = 0;
		do {
			System.out.println("Control = " + control);
			control++;
		}while(control<3);
		
		//for
		
		for (int i = 0; i < 3; i++) {
			System.out.println("Control = " + i);
		}
		
		//break & continue
		
		for(int j = 0; j < 10; j++) {
			if (j % 2 == 0) {
				System.out.println("Control = " + j);
				break;
			}
			System.out.println("Control = " + j);
		}
	}
}
